dpdata/abacus/md.py

```diff
@@ -33,8 +33,17 @@ def get_coord_dump_freq(inlines):
 def get_coords_from_dump(dumplines, natoms):
     nlines = len(dumplines)
     total_natoms = sum(natoms)
-    nframes_dump = int(nlines/(total_natoms + 13))
-    
+    calc_stress = False
+    if "VIRIAL" in dumplines[6]:
+        calc_stress = True
+    else:
+        assert("POSITIONS" in dumplines[6] and "FORCE" in dumplines[6])
+    nframes_dump = -1
+    if calc_stress:
+        nframes_dump = int(nlines/(total_natoms + 13))
+    else:
+        nframes_dump = int(nlines/(total_natoms + 9))
+    assert(nframes_dump > 0)
```

Comment:
Could you please add explanations for the assertion when it fails? `assert nframes_dump > 0 , "The reason is xxx"`

Reply:
Also, in `assert("POSITIONS" in dumplines[6] and "FORCE" in dumplines[6])`

Reply:
Thanks for the valuable suggestion! I have added assertion failed reason for each case in the abacus interface.